sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/AuthenticationEventResponseHandler.cs

```diff
@@ -157,7 +172,7 @@ internal static AuthenticationEventResponse ConvertToEventResponse(Authenticatio
 
             return responseType.BaseType.GetGenericTypeDefinition() == typeof(ActionableResponse<>) ||
                    responseType.BaseType.GetGenericTypeDefinition() == typeof(ActionableCloudEventResponse<>) ?
-                     (AuthenticationEventResponse)JsonSerializer.Deserialize(response.ToString(), responseType, GetSerializerOptions()) :
+                     (AuthenticationEventResponse)System.Text.Json.JsonSerializer.Deserialize(response.ToString(), responseType, GetSerializerOptions()) :
```

Comment:
This type uses both `System.Text.Json` and `Newtonsoft.Json`. Why?

Reply:
I will use newtonsoft for this. Originally we were using System.Text.Json, but now that we're using newtonsoft in other places, I will change this serializer as well.

Reply:
If you are using System.Text.Json to process the JSON payload, using an exception type from a different package won't work, even if the exceptions have the same name. Long story short, don't use Newtonsoft exception types to handle errors from System.Text.Json. I think the STJ docs recommend catching JsonException when the JSON payload is invalid.

Reply:
We are using STJ in this line, and the newtonsoft error is being thrown from the helper class. A different method below will call this helper class and could throw a newtonsoft error

Reply:
Removed NewtonSoft dependencies